frontend/src/metabase-lib/types.ts

```diff
@@ -17,64 +17,72 @@ import type { ColumnExtractionTag } from "./extractions";
  * An "opaque type": this technique gives us a way to pass around opaque CLJS values that TS will track for us,
  * and in other files it gets treated like `unknown` so it can't be examined, manipulated or a new one created.
  */
-declare const Query: unique symbol;
-export type Query = unknown & { _opaque: typeof Query };
+declare const QuerySymbol: unique symbol;
```

Comment:
All changes in this file are needed to make `api-extractor` working.
He complains about the case when a file has 2 entities of the same type declared, in our case a `const` and a `type`

Reply:
That is very interesting, I thought `const` declares a type for a constant in the file and `type` declare a type, so those aren't really the same thing. Likely a bug in API Extractor 😅

Reply:
Btw the issue that helped me debugging the error, because the error itself does not contain any helpful info:
```
InternalError: Internal Error: _notifyReferencedAstEntity() called after analysis is already complete

You have encountered a software defect. Please consider reporting the issue to the maintainers of this application.

```

https://github.com/microsoft/rushstack/issues/1108